### DISCUSSION TITLE
Monitor resource usage by a simulation

### DISCUSSION THREAD

**Discussion by marsluca** [omnetpp/omnetpp#1213] [Q&A]
I have developed five scheduling algorithms using OMNeT and INET. Is there a way to generate a graph showing which one is more resource-intensive? For example, RAM and CPU usage (other misutations are also fine). Any possible solution will be appreciated.

**Comment by rhornig** [comment]
It's not clear what you want to measure:
1. the resource usage of the SIMULATION MODEL of the scheduling algorithm (i.e. how fast the simulation runs)
2. or the resource usage of the actual algorithm in real-life

the first can be monitored by the opsystem's resource monitoring tools 
the second cannot be measured by an INET model, as INET simulates the network communication and does not simulate resource usage of the algorithms
